templates/layouts/mtk-install.hbs

```diff
@@ -43,6 +43,7 @@
         <p>
           <a class="btn btn-primary" href="https://release.asteroidos.org/nightlies/{{.}}/asteroid-image-{{.}}.ext4" role="button">asteroid-image-{{.}}.ext4</a>
           <a class="btn btn-primary" href="https://release.asteroidos.org/nightlies/{{.}}/zImage-dtb-{{.}}.fastboot" role="button">zImage-dtb-{{.}}.fastboot</a>
+          <a class="btn btn-primary" href="https://release.asteroidos.org/nightlies/{{.}}/SHA512SUMS" role="button">SHA512SUMS</a>
```

Comment:
ehehe, i was actually wondering if might be that easy. Thanks for implementing so very swiftly.